### PR TITLE
notification for newer versions

### PR DESCRIPTION
**Pull request by yzainee**

### CODE REVIEW

File: f8a_notification/entryscript.py [hunk 1]
```diff
@@ -14,14 +14,286 @@
...
+                            "id": str(uuid4()),
+                            "type": "notifications"
+                        }
+                        }
+            tmp_json['data']['attributes']['custom']['repo_url'] = data
+            tmp_json['data']['attributes']['id'] = data
+            tmp_json['data']['attributes']['custom']['version_updates'] \
+                = repo_data['version_updates']
+            final_payload.append(tmp_json)
+    print("<---Repo Data--->")
```
Comment (GeetikaBatra) [inline f8a_notification/entryscript.py]: These print statements should not be a part of the code, I believe.

File: f8a_notification/entryscript.py [hunk 2]
```diff
@@ -14,14 +14,286 @@
...
+
+
+def read_packages():
+    """Read all the packages last updated."""
+    prev_date = (datetime.utcnow() - timedelta(1)).strftime('%Y%m%d')
+    query_str = "g.V().has('latest_version_last_updated',prev_date).valueMap()"
+    # prev_date = '20180805' for testing purpose, change date here
+    payload = {
+        'gremlin': query_str,
+        'bindings': {
```
Comment (msrb) [inline f8a_notification/entryscript.py]: Nice! I didn't know this is possible. We do it "the weird way" in other services.

File: openshift/f8a-notification.yaml [hunk 3]
```diff
@@ -22,6 +22,15 @@ objects:
...
             containers:
+            env:
+            - name: BAYESIAN_GREMLIN_HTTP_SERVICE_HOST
+              value: bayesian-gremlin-http
+            - name: BAYESIAN_GREMLIN_HTTP_SERVICE_PORT
+              value: "8182"
+            - name: LICENSE_SERVICE_HOST
+              value: f8a-license-analysis
+            - name: LICENSE_SERVICE_PORT
+              value: "6162"
```
Comment (GeetikaBatra) [inline openshift/f8a-notification.yaml]: @yzainee Next two lines should be outside of env scope.

**Comment by GeetikaBatra** [comment]
Merging this one as these changes are needed for further work.

File: f8a_notification/entryscript.py (hunk 2 above)
Comment (miteshvp) [reply f8a_notification/entryscript.py]: Since very long - https://github.com/fabric8-analytics/f8a-server-backbone/blob/master/src/recommender.py#L139

**Pull request merged**